### PR TITLE
Init

### PR DESCRIPTION
**Pull request by niekcandaele**
# High level change log

- Main latex template/layout
- Written down requirements of project + ideas toward implementation
- Create sections for different tools I'd like to explore
- Script to generate week reports automatically ( :P )

I moved the latex from the root of the repo to a subfolder, which ruined some diffs, sorry :(

CI will build a PDF for any commit where the latex changes, for this PR you can find it [here](https://github.com/niekcandaele/BAP/actions/runs/581980795).

### CODE REVIEW

**Comment by StampixSMO** [comment]
Wrt  1.3.3
Note that campaign validity configuration such as length, amount of photos, valid codes, ... is database configuration that is put into use by calling the API from the web app.

**Comment by niekcandaele** [comment]
> Wrt 1.3.3
> Note that campaign validity configuration such as length, amount of photos, valid codes, ... is database configuration that is put into use by calling the API from the web app.

Right, forgot!

Can you give some other fields that should be in that deployment configuration? We talked about styling stuff (which can be done with Storybook).

If it's a lot of fields, the CMS-deployment-config thing (or similar) is viable. If it's just a few fields, they could all live in the database and can be made accessible via the admin panel/api.

**Comment by StampixSMO** [comment]
@niekcandaele CTRL+Shift+F in the repo for `Theme.instance.current.`

BTW, don't forget about i18n, that's also in Contentful right now.

File: thesis/Chapters/Chapter1.tex [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+\subsection{Deployment configuration}
+
+All the following is currently stored inside the CMS but define a configuration of the deployment.
+
+\begin{itemize}
+	\item Domain name (company.stampix.com).
+	\item Campaign ID (corresponds to an ID in the database)
+	\item Demo mode (No real codes needed but nothing gets printed either)
+	\item googleAnalyticsTrackingCode
+	\item facebookPixelId
```
Comment (StampixSMO) [inline thesis/Chapters/Chapter1.tex:70]: There's lots more btw :)

File: thesis/Chapters/Chapter1.tex [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+TODO: How will this be configured? Could be a multi-select in the deployment configuration
+
+\subsubsection{NSFW check}
+
+This plugin checks every order for photos that has \textbf{N}ot \textbf{S}afe \textbf{F}or \textbf{W}ork content. If an order includes content like this, it gets rejected.
+
+\subsubsection{Thank you module}
+
+After creating an order, the user is presented with a small form that asks if they would like to say thank you. The user can select an emoji to reflect their feelings. 
+This module is particularly useful for creating metrics at the end of a campaign
```
Comment (StampixSMO) [inline thesis/Chapters/Chapter1.tex:101]: Same here, lot's of stuff that's already defined, lmk if I can help out with that

File: thesis/Chapters/Chapter2.tex [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+\subsection{Terraform}
+
+Terraform \footnote{https://www.terraform.io/} is a infrastructure as code tool. In contrast to Ansible, 
+Terraform focuses mostly on infrastructure tasks like creating virtual machines or allocating cloud resources.
+
+\subsection{Serverless}
+\subsection{AWS Cloud Development Kit}
+\subsection{Cloudformation}
+
+// Cloudformation is kind of a legacy thing vs AWS CDK but probably we should still include it?
```
Comment (StampixSMO) [inline thesis/Chapters/Chapter2.tex:105]: The CDK still transpiles to Cloudformation (or Terraform), CDK is just an abstraction layer

**Pull request merged**